### ISSUE TITLE
[UncappedChances] Melee Attacks Dealing Too Much Damage

### ISSUE THREAD

**Issue by Cooltank10** [ZetaDaemon/RoR2Mods#1]
Upon starting a new game and getting hit from a melee attack, it deals much more damage (Around x^2) than their default base dmg. Beetles deal 144 dmg instead of their base 12 dmg. Lemurians deal 144 dmg instead of 24 dmg from the 200% dmg melee attack. I've tested all my mods and this one is the only one causing this issue.

**Comment by ZetaDaemon** [comment]
Is this on the latest update of the game? the mod has not been updated since the survivors of the void update so issues were expected
I do plan on eventually updating the mod had just sorta been waiting on gearbox fixes, my b for not getting back to it sooner

**Comment by Cooltank10** [comment]
Yes, it is the latest update. Just letting you know that there are problems, I know being a Dev is hard. :)

**Comment by Robo-Games** [comment]
I'm not having this issue, it may be a compatibility issue with a different mod? I know you said you tested all the mods, but it may be this mod interacting with a different mod

**Comment by Cooltank10** [comment]
I'll try the mod by itself and test if the problem still exists. Btw, thanks for replying quickly! :)
